cluster_assign_roles.rb

```diff
@@ -285,6 +293,58 @@ def install_kafka(target_nodes)
     end
   end
 
+  def permission_host(node)
```

Comment:
Reference for the recommended way to use the old chef-server behaviour that we rely on:

https://blog.chef.io/2014/11/10/security-update-hosted-chef/

Reply:
Thanks; we should file this as an issue unless this will not work to get us off the ground?

Reply:
Issue #1240 submitted